helm/README.md

```diff
@@ -165,8 +165,10 @@ The deployment is configured via values.yaml file.
 | maxBlocks | Maximum number of worker pods to spawn | 100 |
 | maxWorkersPerPod | How many workers will be scheduled in each pod | 1 |
 | endpointName | (Optional) Specify a name for registration with the funcX web services | The release name (Release.Name) |
-| endpointUUID | (Required) Specify a UUID for this endpoint | |
 | endpointCLIargs | Any additional command line arguments to give to the `funcx-endpoint` executable | |
+| detachEndpoint | Run the endpoint as a daemon inside the pod? | true |
+| endpointUUID   | (Required) Specify a UUID for this endpoint. | |
+| endpointCLIargs | Any additional command line arguments to give to the `globus-compute-endpoint` executable | |
 | maxIdleTime  | The maximum time to maintain an idle worker. After this time the SimpleStrategy will terminate the idle worker. | 3600 |
```

Comment:
- Doh; looks like the funcx-endpoint didn't get removed.  (`endpointCLIargs` is now duplicated).
- `endpointUUID` gained two extra spaces.

Reply:
Should be fixed